### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- fix bug can not click under noti.
image
- fix bug import [here](https://www.notion.so/kybernetwork/All-network-Swap-Automatically-show-the-importing-default-token-on-swap-page-5e7cd0ea4a0143be98032a799549167e)

### CODE REVIEW

File: src/pages/SwapV2/index.tsx [hunk 1]
```diff
@@ -624,6 +626,7 @@ export default function Swap({ history }: RouteComponentProps) {
 
   useEffect(() => {
     checkAutoSelectTokenFromUrl()
+    initialTotalTokenDefault.current = Object.keys(defaultTokens).length
```
Comment (hungdoansy) [inline src/pages/SwapV2/index.tsx:629]: it can be moved outside of this effect

**Review by viet-nv — APPROVED** [review]

File: src/pages/SwapV2/index.tsx (hunk 1 above)
Comment (namgold) [reply src/pages/SwapV2/index.tsx:629]: What if @nguyenhoaidanh indent for this value not update realtime but only after some deps updated? In this case it is only first time initially (no deps)

Comment (namgold) [reply src/pages/SwapV2/index.tsx:629]: Should have some explain from @nguyenhoaidanh

Comment (hungdoansy) [reply src/pages/SwapV2/index.tsx:629]: Ah yeah, my bad. An explanation here is good

Comment (nguyenhoaidanh) [reply src/pages/SwapV2/index.tsx:629]: yes, Nam is correct. I named it `initial`TotalTokenDefault.     I just want to get init total count of default token.

**Pull request merged**